### DISCUSSION TITLE
Are `stdout_pretty_tests_report` and `colour_report` mutually exclusive?

### DISCUSSION THREAD

**Discussion by matrdr** [ThrowTheSwitch/Ceedling#802] [Q&A] [answered]
I like the colour report for adding some colour that makes failed and succeeded tests immediately visible. But at the same time I appreciate the _Overall Test Summary_ provided by the `stdout_pretty_tests_report` plugin.

It seems if you activate both, you get the test report, but not the colours.

Do I assume correctly that these plugins do not work together?

**Comment by mvandervoord** [comment, marked as answer]
You are correct. While you can have multiple stored reports enabled, only one stdout report can be enabled at a time.

**Reply by matrdr** [reply]
Thank you for the clarification. It is a pity, but I can live with that. I should probably just make two configurations. One with colour output when I am only running one test file and the other one when I run many tests at once.

**Reply by mkarlesky** [reply]
@matrdr I'm sorry to say that we had to yank the color report plugin for the upcoming 1.0.0 release. It was based on some dodgy hooks, and the matching scheme was too brittle to work with other changes for the upcoming release. Lots of work has happened in logging, though, so that we can bring back something that works well. In the meantime, we have added some fun emoji to the logging output (it can be disabled).

**Reply by matrdr** [reply]
Haha, no problem! Emoji output sounds cool, too. Looking forward for the release! 😊

**Comment by mvandervoord** [comment]
It sounds like we should be updating this behavior to at least support a nice summary in the color output, or color in the pretty output :)
